core/components/atoms/switchInput/__stories__/index.story.tsx

```diff
@@ -19,7 +19,7 @@ export const all = () => {
 
   return (
     <div>
-      <Switch disabled={disabled} checked={checked} size={size} onChange={onChangeHandler} />
+      <Switch aria-label="Switch" disabled={disabled} checked={checked} size={size} onChange={onChangeHandler} />
```

Comment:
Can we add an example with Label too? with `htmlFor` and `id`

<Label>Have you been vaccinated?</Label>
<Switch>